client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/RemoteQueryStringTest.java

```diff
@@ -131,6 +131,23 @@ protected void initProtoSchema(RemoteCacheManager remoteCacheManager) throws IOE
       serCtx.registerMarshaller(new AnalyzerTestEntityMarshaller());
    }
 
+
+   /**
+    * Logs the Protobuf schema errors (if any) and fails the test appropriately.
+    */
+   protected void checkSchemaErrors(RemoteCache<String, String> metadataCache) {
```

Comment:
This code is a duplicate of the same in https://github.com/infinispan/infinispan/pull/5439/files#diff-0cd17a9e63616a6e9c7fc4a6f83db184R123
Can it be reused ?

Reply:
I'm not sure where I could move that code. ProtobufMetadataManager does not seem like a good fit.